exporter/collector/logs.go

```diff
@@ -124,6 +124,27 @@ var otelSeverityForText = map[string]plog.SeverityNumber{
 	"fatal4": plog.SeverityNumberFatal4,
 }
 
+type AttributeProcessingError struct {
```

Comment:
Does this need to be public?  Same question for other error below.

Reply:
I tend to make these public out of habit so that they can be used in test assertions, but since we don't use the `_test` package for these tests I can make these private.

Reply:
Done in [4b7052e](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/794/commits/4b7052e0d2962cb46c613a4f1e56d431b7ad894e)